cmd/src/api.go

```diff
@@ -123,8 +127,25 @@ func gqlURL(endpoint string) string {
 	return endpoint + "/.api/graphql"
 }
 
+// authenticationHeader returns the header name and header value required to authenticate a request,
+// or `"", ""` if one calls this function when the config has no AccessToken information
+func authenticationHeader(cfg *config) (string, string) {
+	if cfg.AccessToken == "" {
+		return "", ""
+	}
+
+	var headerValue string
+	if cfg.RunAs != "" {
+		headerValue = fmt.Sprintf(`token-sudo user="%s",token="%s"`, cfg.RunAs, cfg.AccessToken)
+	} else {
+		headerValue = "token " + cfg.AccessToken
+	}
+
+	return "Authorization", headerValue
```

Comment:
nit: Remove blank line before return

Reply:
For my curiosity, is this a Sourcegraph coding convention? I was sad that `go fmt ./...` didn't say anything about it

Reply:
Hmm, I don't think there is anything formal stating this anywhere. And gofmt doesn't have any formal rules about how code should be separate by whitespace AFAIK. You could e.g. have no blank lines in a function and gofmt wouldn't complain, or you could have tons and likewise it wouldn't complain AFAIK :P

My general rule of thumb is that if the return statement is a simple 1 liner, then no whitespace separating it from the code block above is useful.

You may also appreciate this site if you haven't seen it: https://dmitri.shuralyov.com/idiomatic-go